lib/order_manager.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'menu'
+
+class OrderManager
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+  end
+
+  def show_menu
+    @menu.show_dishes
+  end
+
+  def select(dishes)
+    dishes.each do |dish, quantity|
```

Comment:
RSpec gave me this error here:

```
NoMethodError:
       undefined method `each' for "pad thai":String
```

What types of object can have the method `each` called on them?